distrobuilder/main.go

```diff
@@ -548,6 +548,8 @@ func addSystemdGenerator() {
 # NOTE: systemctl is not available for systemd-generators
 set -eu
 
+export LC_ALL=C
```

Comment:
We will want a comment here explaining why its here so it doesn't get inadvertently removed